pkg/build/api/v1/types.go

```diff
@@ -626,6 +691,17 @@ type GenericWebHookEvent struct {
 type GitInfo struct {
 	GitBuildSource    `json:",inline"`
 	GitSourceRevision `json:",inline"`
+
+	// Refs is a list of GitRefs for the provided repo - generally sent
+	// when used from a post-receive hook. This field is optional and is
+	// used when sending multiple refs
+	Refs []GitRefInfo `json:",omitempty"`
```

Comment:
i don't need it, the problem is that types https://github.com/openshift/origin/blob/master/pkg/build/api/types.go#L670 is inconsistent with https://github.com/openshift/origin/blob/master/pkg/build/api/v1/types.go#L626  and https://github.com/openshift/origin/blob/master/pkg/build/api/v1beta3/types.go#L600 . So every time i use *GitInfo in my githubwebhookinfo or the generic one, regardless it starts barking. Should it be like this or is this an error?